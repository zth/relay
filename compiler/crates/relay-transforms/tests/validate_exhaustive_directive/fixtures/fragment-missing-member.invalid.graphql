#expected-to-throw
fragment FragmentMissingMember on UserNameRenderer @exhaustive {
  ... on PlainUserNameRenderer { __typename }
  ... on MarkdownUserNameRenderer { __typename }
}