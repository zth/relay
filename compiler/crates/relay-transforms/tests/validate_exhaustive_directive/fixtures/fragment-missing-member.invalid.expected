==================================== INPUT ====================================
#expected-to-throw
fragment FragmentMissingMember on UserNameRenderer @exhaustive {
  ... on PlainUserNameRenderer { __typename }
  ... on MarkdownUserNameRenderer { __typename }
}
==================================== ERROR ====================================
✖︎ Fragment 'FragmentMissingMember' marked with @exhaustive is missing selection for union members: 'CustomNameRenderer'.

  fragment-missing-member.invalid.graphql:2:10
    1 │ #expected-to-throw
    2 │ fragment FragmentMissingMember on UserNameRenderer @exhaustive {
      │          ^^^^^^^^^^^^^^^^^^^^^
    3 │   ... on PlainUserNameRenderer { __typename }
